### ISSUE TITLE
[json-home] content-type with "+ convention"

### ISSUE THREAD

**Issue by RubenVerborgh** [mnot/I-D#172]
I would have expected `application/home+json` in line with the "+ convention".

**Comment by asbjornu** [comment]
That would also be more in line with [RFC 7807](https://tools.ietf.org/html/rfc7807)'s `application/problem+json`.

**Comment by mnot** [comment]
Yes. I've intentionally not registered the media type because I want the flexibility to change the format when it's more baked.

**Comment by dret** [comment]
as one additional data point: https://tools.ietf.org/html/draft-wilde-home-xml-03#section-4.1 defines `application/problem+xml` assuming that the structured syntax suffix is a useful pattern to follow.

**Comment by RubenVerborgh** [comment]
Seems like it's time to turn this convention into an officially meaningful thing.

**Comment by dret** [comment]
@RubenVerborgh: it is an RFC (https://tools.ietf.org/html/rfc6838), but media types are free to follow or ignore this pattern.
